src/main/groovy/edu/oregonstate/mist/locations/resources/MergeUtil.groovy

```diff
@@ -135,7 +134,7 @@ class MergeUtil {
      */
     List<ResourceObject> populate(List<ResourceObject> data) {
         data.each {
-            if (it?.attributes?.bldgID == 'VLib') {
+            if (it?.attributes?.abbreviation == 'VLib') {
```

Comment:
bldgID is more reliable than abbreviation. We should be checking for the bldgID of the library, which is 0036.

Reply:
Done.